src/ripple/protocol/Protocol.h

```diff
@@ -53,6 +57,35 @@ std::size_t constexpr dirNodeMaxEntries = 32;
 /** The maximum number of pages allowed in a directory */
 std::uint64_t constexpr dirNodeMaxPages = 262144;
 
+/** The maximum number of items in an NFT page */
+std::size_t constexpr dirMaxTokensPerPage = 32;
+
+/** The maximum number of owner directory entries for account to be deletable */
+std::size_t constexpr maxDeletableDirEntries = 1000;
+
+/** The maximum number of token offers that can be canceled at once */
+std::size_t constexpr maxTokenOfferCancelCount = 500;
+
+/** The maximum number of offers in an offer directory for NFT to be burnable */
+std::size_t constexpr maxDeletableTokenOfferEntries = 500;
+
+/** The maximum token transfer fee allowed.
+
+    Token transfer fees can range from 0% to 50% and are specified in tenths of
+    a basis point; that is a value of 1000 represents a transfer fee of 1% and
+    a value of 10000 represents a transfer fee of 10%.
+
+    Note that for extremely low transfer fees values, it is possible that the
+    calculated fee will be 0.
+ */
+std::uint16_t constexpr maxTransferFee = 50000;
+
+/** The maximum length of a URI inside an NFT */
+std::size_t constexpr maxTokenURILength = 512;
```

Comment:
This `512` limit does not match the spec or the documentation, which sets the limit at `256`.  This value should be changed to `256`.